tests/unit/test_scrape_jobs.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+import unittest
+from unittest.mock import PropertyMock, patch
+
+from alertmanager import WorkloadManager
+from charm import AlertmanagerCharm
+from helpers import k8s_resource_multipatch
+from ops.testing import Harness
+
+
+class TestWithInitialHooks(unittest.TestCase):
+    container_name: str = "alertmanager"
+
+    @patch("lightkube.core.client.GenericSyncClient")
+    @patch.object(WorkloadManager, "check_config", lambda *a, **kw: ("ok", ""))
+    @k8s_resource_multipatch
+    @patch.object(WorkloadManager, "_alertmanager_version", property(lambda *_: "0.0.0"))
+    def setUp(self, *unused):
+        self.harness = Harness(AlertmanagerCharm)
+        self.addCleanup(self.harness.cleanup)
+
+        self.harness.set_leader(True)
+        self.app_name = "alertmanager-k8s"
```

Comment:
Shorter is nicer to use in string literals.

```suggestion
        self.app_name = "am"
```